delivery_postlogistics/postlogistics/web_service.py

```diff
@@ -190,7 +190,10 @@ def _prepare_attributes(
 
         if not services:
             raise exceptions.UserError(
-                _("No Postlogistics packaging services found in this picking.")
+                _(
+                    "No PostLogistics packaging services found "
+                    "in packaging {} of picking {}."
```

Comment:
```suggestion
                    "in packaging {}, for picking {}."
```
I would have written that instead.

Reply:
fixed